docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -27,9 +27,116 @@ and thus can not "hear" one another directly.
 
 :::
 
-MOBILE Rewards are earned as one "lump sum" by a Wallet with a 5G Hotspot with at least one
-connected, SAS registered, CBRS Small Cell Radio every roughly 24-hour interval. This allows a high
-tolerance for downtime in infrastructure that is beyond the 5G Hotspot Owner's control.
+---
+
+## How To Mine MOBILE
+
+Several factors contribute to the amount of MOBILE a 5G Hotspot will earn over a given period. This
+page covers some higher-level concepts, design elements, and rules to keep in mind that will help
+better explain what you might earn and why. This section covers these and some mining and token
+reward basics.
+
+The Helium Blockchain rewards 5G Hotspots for providing 5G wireless coverage and verifying the
```

Comment:
This make it sound as if just 5G Hotspot is sufficient to earn MOBILE. Maybe "5G Hotspots with CBRS Radios".

Reply:
updated